io/disk/lvsetup.py.data/lvsetup.yaml

```diff
@@ -0,0 +1,31 @@
+setup:
```

Comment:
Although this yaml file should work fine, I don't see it that important to create a category for each param. If there are no useful values then I don't think you even need to create one. And if you still want to add example, then it might be simplified. For example:

```yaml
disk: None
vg_name: avocado_vg
lv_name: avocado_lv
size: !mux
    small:
        lv_size: "1M"
    big:
        lv_size: "1G"
...
```

In a real life I'd imagine even simpler yaml file:

```yaml
scenario: !mux
    loopback:
        disk: None
        lv_size: "10M"
    real:
        disk: /dev/sde
        lv_size: None
```

rather than multiplexing all possible variants. Yes, one could do it, that doesn't mean he has to do it. Anyway I leave this on your consideration, it's meant to be an example, so I'm fine with either choice.